client/components/App.js

```diff
@@ -1,6 +1,12 @@
 import React, { PropTypes, Component } from 'react';
 import TextArea from './TextArea.js';
-import Button from './button.js';
+import PrevButton from './prevButton.js';
+import NextButton from './nextButton.js';
+import Background from './background.js';
```

Comment:
Just ally opinion, but I think file name should be exactly the same as the component name just to avoid errors in casing